content/en/network_monitoring/cloud_network_monitoring/guide/detecting_application_availability.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Detecting Application Availability using Network Insights
+description: Use CNM to detect application availability
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/apm-cnm-application-debugging/'
+      tag: 'Blog'
+      text: 'Debug application issues with APM and Cloud Network Monitoring'
+    - link: 'https://www.datadoghq.com/blog/cnm-best-practices/'
+      tag: 'Blog'
+      text: 'Best practices for getting started with Datadog Cloud Network Monitoring'
+    - link: 'https://www.datadoghq.com/blog/monitor-containers-with-cnm/'
+      tag: 'Blog'
+      text: 'How to monitor containerized and service-meshed network communication with Datadog CNM'     
+---
+
+## Overview
+
+Modern applications are built in a modular distributed manner with microservices and stateless design principles. Networking connects the distributed components of an application, ensuring a seamless user and application experience. By capturing, analyzing, and correlating network metrics such as latency, packet loss, and throughput across different applications and services, Datadog's Cloud Network Monitoring (CNM) provides actionable insights for application developers and SRE teams.
+
+## Service discovery and connectivity
+
+CNM is designed to track traffic between entities and determine which resources are communicating and their health status.
+
+To examine the a basic traffic flow between entities, use the following steps:
+
+1. On the [Network Analytics page][1], set your **View clients as** and **View servers as** dropdown filters to group by `service` tags to examine a service-to-service flow, and observe the basic traffic unit: a source IP communicating over a port to a destination IP on a port.
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_service_service.png" alt="CNM analytics page, grouping by service to service with Client and Server IP highlighted">}}
+
+   Each row aggregates 5 minutes worth of connections. While you might recognize some IPs as specific addresses or hosts depending on your network familiarity, this becomes challenging with larger, more complex networks. Therefore, the most meaningful aggregation level is rolling up the host or container associated with these IPs to tags in Datadog, such as `service`, `availability zone`, `pod`, and so on.
+
+2. For example, to view the network traffic for all of your `orders-sqlserver*` pods by host and availability zone, use the following filters to narrow down your search results:
```

Comment:
The "for example" here changes the premise of the numbered list - it started out as a procedure to presumably examine any basic traffic flow, but now it's abruptly turned into a hypothetical. These two purposes are different enough that I wouldn't put them in the same list, or I'd be upfront about this being an example from the beginning.

Alternatively, if the point of this step is *actually* narrowing down search results, rather than the example, you could write:
```suggestion
2. Narrow down your search results using filters. For example, to view the network traffic for all of your `orders-sqlserver*` pods by host and availability zone, use the `client_pod_name:orders-sqlserver*` filter:
```